integration-tests/hibernate-orm-panache/src/test/java/io/quarkus/it/panache/PanacheFunctionalityTest.java

```diff
@@ -76,11 +76,15 @@ public void testJaxbAnnotationTransfer() {
     }
 
     /**
-     * This test is disabled in native mode as there is no interaction with the quarkus integration test endpoint.
+     * _PanacheEntityBase_ has the method _isPersistent_. This method is used by jackson to serialize the attribute *persistent*
+     * in the JSON which is not intended. This test ensures that there something done that this does not happen.
+     * 
+     * This test does not interact with the quarkus application itself. It is just using the jackson ObjectMapper with a
+     * PanacheEntity. Thus this test is enabled in native mode. The test code runs the JVM and not native.
      */
     @DisabledOnNativeImage
     @Test
-    public void jacksonDeserilazationIgnoresPersitantAttribute() throws JsonProcessingException {
+    public void JacksonDeserializationIgnoresPersistentAttribute() throws JsonProcessingException {
```

Comment:
All other test methods start with lower-case so we should follow the same rule.

Reply:
Thank you! I was copy pasting in order to make no typos...

Reply:
:+1: